q2_cutadapt/plugin_setup.py

```diff
@@ -32,7 +32,6 @@
 import q2_cutadapt._demux
 import q2_cutadapt._trim
 
-
```

Comment:
**Lint:**
This newline is also on purpose :)

![image](https://user-images.githubusercontent.com/3976804/184199359-6d8f9506-d1a3-46e1-91c3-2b113a5d46a7.png)